+nosnoc/+model/Pds.m

```diff
@@ -0,0 +1,43 @@
+classdef Pds < nosnoc.model.Base
+    properties
+        f_x
+        c
+        E
```

Comment:
Is this matrix already used and mature, or is it unpublished research?

Reply:
I guess this is unpublished research. It is  practically quite robust, however theory remains to be done.

Reply:
Hence, it is also undocumented. I prefer to keep it out of `main`, except it is needed for our current examples. 
But a few lines of comments explaining what it does (to help me and others) would be very welcome!

Reply:
We could alternatively drop the example that needs it (the sweeping set example). I have no strong opinions either way at the moment.

Reply:
Just add a small  comment in the src for what is `E` used and when it is needed.
But I would make `E` an optional thing you pass. E.g., if it is a "regular" PDS, then E is default identity, and you dont even need to pass it.